bedrock/products/templates/products/relay/landing.html

```diff
@@ -26,3 +26,133 @@
 {% block sub_navigation %}
   {% include 'products/relay/includes/subnav.html' %}
 {% endblock %}
+
+
+{% block content %}
+
+<div class="t-hero">
+
+  {% call split(
+    block_class='mzp-l-split-pop mzp-l-split-hide-media-on-sm-md',
+    theme_class='',
+    media_class='',
+    media_include='products/relay/includes/phone.html'
+  ) %}
+    <h1 class="mzp-u-title-md">{{ ftl('hero-section-title') }}</h1>
+    <p>{{ ftl('hero-section-body') }}</p>
+    <p><button>{{ ftl('hero-section-cta') }}</button></p>
+    <img class="c-brands" src="{{ static('img/products/relay/landing/brands.svg') }}" alt="" width="480" height="54">
+  {% endcall %}
+
+  {% include 'products/relay/includes/bundle.html' %}
+
+</div>
+
+<section class="mzp-l-content c-works">
+  <h2 class="mzp-c-section-heading">{{ ftl('how-it-works-section-title') }}</h2>
+  <div class="mzp-l-columns mzp-t-columns-three">
+    <div class="c-works-item">
+      <img src="{{ static('img/products/relay/landing/how/how-it-works-1.svg') }}" alt="" width="306" height="223" loading="lazy" class="c-works-img">
+      <h3 class="mzp-u-title-2xs">{{ ftl('how-it-works-section-extension-headline') }}</h3>
+      <p>{{ ftl('how-it-works-section-extension-body') }}</p>
+    </div>
+    <div class="c-works-item">
+      <img src="{{ static('img/products/relay/landing/how/how-it-works-2.svg') }}" alt="" width="306" height="223" loading="lazy" class="c-works-img">
+      <h3 class="mzp-u-title-2xs">{{ ftl('how-it-works-section-forward-headline') }}</h3>
+      <p>{{ ftl('how-it-works-section-forward-body') }}</p>
+    </div>
+    <div class="c-works-item">
+      <img src="{{ static('img/products/relay/landing/how/how-it-works-3.svg') }}" alt="" width="306" height="223" loading="lazy" class="c-works-img">
+        <h3 class="mzp-u-title-2xs">{{ ftl('how-it-works-section-manage-headline') }}</h3>
+        <p>{{ ftl('how-it-works-section-manage-body') }}</p>
+    </div>
+  </div>
+</section>
+
+<div class="t-odd">
+  <section class="mzp-l-content">
+
+    <h2 class="u-hidden">{{ ftl('landing-reviews-title') }}</h2>
+
+    <div class="c-reviews">
+      <div>
+        <a class="c-review-amo" href="https://addons.mozilla.org/en-US/firefox/addon/private-relay/">
+          <span class="c-review-amo-browser">Firefox Browser</span>
```

Comment:
The wordmark "Firefox Browser" is still displaying so I can't use the default helpers BUT I should absolutely be using an `logo-word-ver-white` here instead of displaying the wordmark as text on the page.